src/main/java/seedu/address/logic/parser/FindCommandParser.java

```diff
@@ -27,7 +27,7 @@ public FindCommand parse(String args) throws ParseException {
 
         String[] nameKeywords = trimmedArgs.split("\\s+");
```

Comment:
A small detail but would be nice to change this to `detailKeywords` for code clarity.

Reply:
Good spot thanks! Feel free to change it in your next PR since this one is closed 😄